openapi/components/schemas/RunListResponse.yaml

```diff
@@ -9,4 +9,8 @@ properties:
   next_page_token:
     type: string
     description: A token which may be supplied as `page_token` in workflow run list request to get the next page of results.  An empty string indicates there are no more items to return.
+  total_runs:
+    type: integer
+    description: >-
+      Total count of workflow runs that the service has executed or is executing and the caller has permission to see.
```

Comment:
Suggestion for brevity, and future-proofing (in case we ever add the ability to filter results from this endpoint). I'd also suggest replacing the words "workflow runs" with just "runs", per the WES terminology.

```suggestion
      Total count of runs in the result set.
```

Reply:
> in case we ever add the ability to filter results from this endpoint

not sure I follow.
The sentence `has executed or is executing` is looks generic in terms of `workflow-execution-service-schemas`.
The sentence `the caller has permission to see` didn't set any constraints in case of policy scope implementation, the records might be across entire data as well.

> "workflow runs" with just "runs", per the WES terminology

As I see in other examples, the `workflow runs` preferred:
![Selection_11021](https://user-images.githubusercontent.com/23525618/189819450-287a55e3-cea6-4423-bc46-7ffc27a771be.jpg)
https://github.com/ga4gh/workflow-execution-service-schemas/blob/develop/openapi/components/schemas/RunListResponse.yaml


![Selection_11022](https://user-images.githubusercontent.com/23525618/189819422-b2c91337-7a91-4f0f-bcb6-d023bdbe7afa.jpg)
https://github.com/ga4gh/workflow-execution-service-schemas/blob/develop/openapi/components/schemas/RunStatus.yaml